src/pages/Sessions.tsx

```diff
@@ -42,50 +130,127 @@ const Sessions = () => {
-              <InputLabel id="session">Session</InputLabel>
-              <Select
-                id="select"
-                label="session"
-                labelId="session"
-                value={currentSessionId}
-                onChange={handleChangeCurrentSessionId}
-              >
-                {sessions.map((session) => (
-                  <MenuItem key={session.id} value={session.id}>
-                    {session.season} {session.year}
-                  </MenuItem>
-                ))}
-              </Select>
-            </FormControl>
-          )}
+    <>
+      <Box display="flex">
+        <Box display="flex" flexGrow={1} alignItems="center">
+          <Box mr={3}>
+            <Typography variant="h1">Session:</Typography>
+          </Box>
+          <Box>
+            {currentSessionId && (
+              <FormControl variant="outlined">
+                <InputLabel id="session">Session</InputLabel>
+                <Select
+                  id="select"
+                  label="session"
+                  labelId="session"
+                  value={currentSessionId}
+                  onChange={handleChangeCurrentSessionId}
+                >
+                  {sessions.map((session) => (
+                    <MenuItem key={session.id} value={session.id}>
+                      {session.season} {session.year}
+                    </MenuItem>
+                  ))}
+                </Select>
+              </FormControl>
+            )}
+          </Box>
         </Box>
+        {currentSessionId && (
+          <>
+            <Box flexShrink={0}>
+              <Button variant="outlined" onClick={handleOpenFormDialog}>
+                Add a client &nbsp;
+                <Add />
+              </Button>
+            </Box>
+            <RegistrationDialog
+              open={displayRegDialog}
+              onClose={handleCloseFormDialog}
+            />
+          </>
+        )}
       </Box>
-      {currentSessionId && (
-        <>
-          <Box flexShrink={0}>
-            <Button variant="outlined" onClick={handleOpenFormDialog}>
-              Add a client
+      <Box mt={4}>
+        <AppBar position="static" className={styles.appbar}>
+          <Tabs
+            TabIndicatorProps={{ style: { backgroundColor: "inherit" } }}
+            value={tab}
+            onChange={(event, value) => {
+              handleTabChange(event, value);
+            }}
+            aria-label="Classes Tabs"
+            // className={styles.tabs}
+          >
+            {/* eslint-disable react/jsx-props-no-spreading */}
+            <Tab value={0} label="All Classes" {...tabProps(0)} />
+            {classes.map((classInfo) => (
+              /* eslint-disable react/jsx-props-no-spreading */
+              <Tab
+                value={classInfo.id}
+                key={classInfo.id}
+                label={classInfo.name}
+                {...tabProps(classInfo.id)}
+              />
```

Comment:
ignore linter rule on only one line

```suggestion
              <Tab
                value={classInfo.id}
                key={classInfo.id}
                label={classInfo.name}
                // eslint-disable-next-line react/jsx-props-no-spreading
                {...tabProps(classInfo.id)}
              />
```